src/main/scala/GeomDataReader.scala

```diff
@@ -0,0 +1,164 @@
+package geotrellis.sdg
+
+import geotrellis.proj4._
+import geotrellis.proj4.util._
+import geotrellis.vector.{Extent, Geometry}
+import geotrellis.vector.reproject._
+import geotrellis.vectortile._
+import geotrellis.layer._
+
+import org.locationtech.geomesa.spark.jts._
+
+import org.locationtech.jts.geom.{Geometry => JTSGeometry}
+
+import org.apache.commons.io.IOUtils
+
+import org.apache.spark.sql.{SQLContext, DataFrame}
+import org.apache.spark.sql.functions.{udf, explode, lit}
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+import java.net.URI
+import java.util.zip.GZIPInputStream
+
+
+object GeomDataReader {
```

Comment:
Warning: Overly general name. 

Geom - geometry of some kind, ok.
Data - What data? Does geometry have the data is the geometry data ?
Reader - Makes sense, we read something from the "targetPath".

"targetPath" - what is it target for, it must be input.

This is setup to specifically read MBTiles files and names and parameters should reflect that to save writing lots of comments explaining it.

Reply:
Understood :+1: